traffic_ops/traffic_ops_golang/crconfig/deliveryservice_test.go

```diff
@@ -35,19 +35,53 @@ import (
 	"gopkg.in/DATA-DOG/go-sqlmock.v1"
 )
 
-func randDS() tc.CRConfigDeliveryService {
+func randDS(ttlOverride int) tc.CRConfigDeliveryService {
 	// truePtr := true
 	falseStrPtr := "false"
 	// numStr := "42"
 	ttlAdmin := "traffic_ops"
-	ttlExpire := "604800"
-	ttlMinimum := "30"
-	ttlRefresh := "28800"
-	ttlRetry := "7200"
-	ttl := util.IntPtr(test.RandInt())
-	ttlStr := strconv.Itoa(*ttl)
-	ttlNS := "3600"
-	ttlSOA := "86400"
+	ttlExpire := 604800
+	ttlMinimum := 30
+	ttlRefresh := 28800
+	ttlRetry := 7200
+	if ttlOverride > 0 {
+		if ttlOverride < ttlExpire {
```

Comment:
Same comment about the `min` function here

Reply:
Using `math.Min()` in f1340e3d56